src/selectors/getChildren.ts

```diff
@@ -95,3 +94,18 @@ export const isChildVisibleWithCursorCheck = _.curry((state: State, path: Path,
   isChildVisible(state, pathToContext(simplePath), { value: pathHeadValue(state, path, child), rank: child.rank }) ||
   isChildInCursor(state, path, simplePath, child)
 })
+
+/** Checks if the child is created after latest absolute context toggle. */
+const isCreatedAfterAbsoluteToggle = _.curry((state: State, child: Child | ThoughtContext) => child.lastUpdated && state.absoluteContextTime && child.lastUpdated > state.absoluteContextTime)
+
+/**
+ * Children filter predicate used for rendering.
+ *
+ * 1. Checks if the child is visible.
+ * 2. Checks if child is within cursor.
+ * 3. Checks if child is created after latest absolute context toggle if starting context is absolute.
+ */
+export const childrenFilterPredicate = _.curry((state: State, path: Path, simplePath: SimplePath, child: Child | ThoughtContext) => {
+  return isChildVisibleWithCursorCheck(state, path, simplePath, child) &&
```

Comment:
Is `isChildVisibleWithCursorCheck` still used externally to this module? If not, we can remove `export` and make it internal.